syz-verifier/main.go

```diff
@@ -136,6 +136,11 @@ func main() {
 		pools[idx] = pi
 	}
 
+	if len(pools) < 2 {
+		printUsage()
```

Comment:
This duplicates what `flag.Usage()` prints and what's being printed for `syz-verifier -help`, and this can get out of sync with the default usage message.
It's better to call `flag.Usage()` here.